Neos.ContentRepository.Core/Classes/SharedModel/Workspace/WorkspaceName.php

```diff
@@ -89,8 +89,7 @@ public static function transliterateFromString(string $name): self
 
         // If the name is still invalid at this point, we fall back to md5
         if (!self::hasValidFormat($name)) {
-            $prefix = 'workspace-';
-            $name = $prefix . substr(md5($originalName), 0, self::MAX_LENGTH - strlen($prefix));
+            $name = substr(md5($originalName), 0, self::MAX_LENGTH);
```

Comment:
Just curiously asking: why did you change that, I kinda liked the `workspace-` prefix?

Reply:
Because the md5 is 32 chars already. And so you would shorten it always.

Reply:
ok, makes sense